internal/store/store.go

```diff
@@ -583,10 +584,11 @@ func (s *Store) GetAllDistroSources(distro string) map[string]SourceConfig {
 
 func NewSourceConfig(repo rpmmd.RepoConfig, system bool) SourceConfig {
 	sc := SourceConfig{
-		Name:    repo.Name,
-		System:  system,
-		RHSM:    repo.RHSM,
-		GPGKeys: repo.GPGKeys,
+		Name:           repo.Name,
+		System:         system,
+		RHSM:           repo.RHSM,
+		GPGKeys:        repo.GPGKeys,
+		ModuleHotfixes: repo.ModuleHotfixes,
```

Comment:
So, this change should have been added to https://github.com/osbuild/osbuild-composer/pull/3831/commits/33a430104b145100b8b89020d3043e26beb7a6d1 instead of https://github.com/osbuild/osbuild-composer/pull/3831/commits/aa6bc9c669b7e61d153bdf28b8b183d52775dcf3, but I won't insist on moving it. Please be more careful next time.

Reply:
ugh, let me fix that.

Reply:
Fixed